components/elm/src/biogeochem/PhenologyMod.F90

```diff
@@ -2559,8 +2559,11 @@ subroutine CropPlantDate (num_soilp, filter_soilp, num_pcropp, filter_pcropp, &
          call calculate_eto(t_ref2m(p), netrad, eflx_soil_grnd(p), forc_pbot(t), forc_rh(t), forc_wind(t), dt, ETout)
          ! monthly ETo
          ETo(p,kmo) = ETo(p,kmo) + ETout
+         
          ! calculate the P:PET for each month
-         p2ETo(p,kmo) = xp(p,kmo)/ETo(p,kmo)
+         if( abs(ETo(p,kmo)) > 0._r8) then 
```

Comment:
- please add whitespace
- Shouldn't there also be code corresponding to an `else` condition?